DigitalLearningSolutions.Web/Views/LearningPortal/Current/MarkActionPlanItemAsComplete.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+@using Microsoft.Extensions.Configuration
+@model SetLearningLogItemCompletionDateViewModel
+@inject IConfiguration Configuration
+
+@{
+  ViewData["Title"] = $"Mark {Model.ResourceName} as complete";

```

Comment:
Need to have the tab title turn to "Error: {title}" when an error occurs.

Reply:
changed